### PR TITLE
Feature/133/deactivation of products

### PR DESCRIPTION
**Pull request by gluhovSemen**

### CODE REVIEW

File: src/apps/inventory/views.py [hunk 1]
```diff
@@ -41,7 +41,18 @@ class ProductBySellerListView(LoginRequiredMixin, ListView):
 
     def get_queryset(self):
         if self.request.user.is_authenticated:
-            return Product.objects.filter(seller=self.request.user)
+            return Product.objects.filter(seller=self.request.user, is_active=True)
+        return Product.objects.none()
```
Comment (volodin28) [inline src/apps/inventory/views.py]: i think we can get rid of if-statement due to LoginRequiredMixin

leave only this
    def get_queryset(self):
        return Product.objects.filter(seller=self.request.user, is_active=True)

File: src/apps/inventory/views.py [hunk 2]
```diff
@@ -41,7 +41,18 @@ class ProductBySellerListView(LoginRequiredMixin, ListView):
...
+
+class DeactivatedProductBySellerListView(LoginRequiredMixin, ListView):
+    model = Product
+    paginate_by = 10
+    template_name = "product_by_seller_list.html"
+
+    def get_queryset(self):
+        if self.request.user.is_authenticated:
+            return Product.objects.filter(seller=self.request.user, is_active=False)
         return Product.objects.none()
```
Comment (volodin28) [inline src/apps/inventory/views.py]: same here

File: src/apps/inventory/views.py (hunk 1 above)
Comment (gluhovSemen) [reply src/apps/inventory/views.py]: fair, just deleted

File: src/apps/inventory/views.py (hunk 2 above)
Comment (gluhovSemen) [reply src/apps/inventory/views.py]: fixed

Comment (gluhovSemen) [reply src/apps/inventory/views.py]: fixed

**Review by egorvavilov — APPROVED** [review]

**Pull request merged**